tests_new/integration_tests/client.py

```diff
@@ -29,9 +29,9 @@ def query(self, query: str):
         graphql_endpoint = os.path.join(os.environ['API_ENDPOINT'], 'graphql', 'api')
         headers = {'AccessKeyId': 'none', 'SecretKey': 'none', 'authorization': self.token}
         r = requests.post(graphql_endpoint, json=query, headers=headers)
-        r.raise_for_status()
         if errors := r.json().get('errors'):
             raise GqlError(errors)
+        r.raise_for_status()
```

Comment:
Irrelevant change to first check for any errors and then raise for status. Sometimes the returned code when there are errors is not 2xx.